### PR TITLE
Create build pipeline

### PR DESCRIPTION
**Pull request by mkopec87**

### CODE REVIEW

**Comment by mkopec87** [comment]
It will build main branch as well as PRs to main - is this enough?

**Comment by adri0** [comment]
This is pretty sweet :)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```
Comment (adri0) [inline .github/workflows/test.yml:16]: In my branch I'm using pattern match, which is a 3.10 exclusive: https://docs.python.org/3/reference/compound_stmts.html#match

Do you think we could keep it only for 3.10+ ?

**Review by adri0 — CHANGES_REQUESTED** [review]

**Comment by adri0** [comment]
> It will build main branch as well as PRs to main - is this enough? 

More than enough for now :)

File: .github/workflows/test.yml (hunk 1 above)
Comment (mkopec87) [reply .github/workflows/test.yml:16]: Fine for me :)

**Review by adri0 — APPROVED** [review]

**Pull request merged**